File: client-module-server/src/main/java/br/edu/les/module/client/domain/Status.java
package br.edu.les.module.client.domain;

import lombok.*;
import org.springframework.stereotype.Component;

import javax.persistence.*;

@Builder
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Component
@Entity
@Table(name = "status")
public class Status extends EntidadeDominio {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "sts_id ")
    private Integer id;

    @Column(name = "sts_status")
    private String nome;
}

File: client-module-server-node/src/models/Status.ts
import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";

@Entity({name: "status"})
class Status {

    @PrimaryGeneratedColumn({name: "sts_id"})
    id: number;

    @Column({name: "sts_status"})
    nome: string;
}

export { Status };